clang++ 3.9 - error: moving a local object in a return statement

```
rapidjson/test/unittest/prettywritertest.cpp:217:12: error: moving a local object in a return statement prevents copy elision [-Werror,-Wpessimizing-move]
    return std::move(writer);
           ^
rapidjson/test/unittest/prettywritertest.cpp:217:12: note: remove std::move call here
    return std::move(writer);
           ^~~~~~~~~~      ~
1 error generated.
```
(same for `rapidjson/test/unittest/writertest.cpp:510:12`)

```
diff --git a/test/unittest/prettywritertest.cpp b/test/unittest/prettywritertest.cpp
index 42ff3f2..13d1a8d 100644
--- a/test/unittest/prettywritertest.cpp
+++ b/test/unittest/prettywritertest.cpp
@@ -214,7 +214,7 @@ static PrettyWriter<StringBuffer> WriterGen(StringBuffer &target) {
     writer.StartObject();
     writer.Key("a");
     writer.Int(1);
-    return std::move(writer);
+    return writer;
 }

 TEST(PrettyWriter, MoveCtor) {
diff --git a/test/unittest/writertest.cpp b/test/unittest/writertest.cpp
index feb4d74..d346e0f 100644
--- a/test/unittest/writertest.cpp
+++ b/test/unittest/writertest.cpp
@@ -507,7 +507,7 @@ static Writer<StringBuffer> WriterGen(StringBuffer &target) {
     writer.StartObject();
     writer.Key("a");
     writer.Int(1);
-    return std::move(writer);
+    return writer;
 }

 TEST(Writer, MoveCtor) {
```

--- a/test/unittest/prettywritertest.cpp
+++ b/test/unittest/prettywritertest.cpp
@@ -214,7 +214,7 @@ static PrettyWriter<StringBuffer> WriterGen(StringBuffer &target) {
     writer.StartObject();
     writer.Key("a");
     writer.Int(1);
-    return std::move(writer);
+    return writer;
 }
 
 TEST(PrettyWriter, MoveCtor) {

--- a/test/unittest/writertest.cpp
+++ b/test/unittest/writertest.cpp
@@ -507,7 +507,7 @@ static Writer<StringBuffer> WriterGen(StringBuffer &target) {
     writer.StartObject();
     writer.Key("a");
     writer.Int(1);
-    return std::move(writer);
+    return writer;
 }
 
 TEST(Writer, MoveCtor) {

